src/main/java/org/broadinstitute/hellbender/engine/MultiVariantDataSource.java

```diff
@@ -60,6 +60,8 @@
     private CloseableIterator<VariantContext> currentIterator;
     private SortedSet<String> mergedSamples;
 
+    private boolean skipDictionaryValidation = false;
```

Comment:
Not sure I have all the context here, but if there are a large amount of gCNV VCFs that have already been generated with inconsistent and/or missing dictionaries, can we just run UpdateVCFSequenceDictionary on them once and pretend like it never happened? I think it would be good to enforce correct behavior with this pipeline (and/or with this engine-level class) going forward.